sync.go

```diff
@@ -28,6 +32,9 @@ func (w *WAL) threadedSync() {
 
 		// Signal waiting threads that they can continue execution
 		w.syncCond.Broadcast()
+
+		// Wait a millisecond to allow for more syncs to queue up
+		<-time.After(time.Nanosecond)
```

Comment:
`time.Sleep`?

Reply:
Wow for some reason that never crossed my mind. I guess I'm a channel fanboy.